pages/instructorFileEditor/instructorFileEditor.js

```diff
@@ -142,13 +144,22 @@ router.get('/*', (req, res, next) => {
                 callback(null);
             } else {
                 debug('Read job sequence');
-                serverJobs.getJobSequence(fileEdit.jobSequenceId, res.locals.course.id, (err, job_sequence) => {
+                serverJobs.getJobSequenceWithFormattedOutput(fileEdit.jobSequenceId, res.locals.course.id, (err, job_sequence) => {
                     if (ERR(err, callback)) return;
                     fileEdit.jobSequence = job_sequence;
                     callback(null);
                 });
             }
         },
+        (callback) => {
+            const promisified = callbackify(editorUtil.getErrorsAndWarningsForFilePath);
+            promisified(res.locals.course.id, relPath, (err, data) => {
+                if (ERR(err, callback)) return;
+                fileEdit.sync_errors = data.errors;
+                fileEdit.sync_warnings = data.warnings;
+                callback(null);
+            });
+        },
```

Comment:
```suggestion
        callbackify(async () => {
            data = await editorUtil.getErrorsAndWarningsForFilePath(res.locals.course.id, relPath);
            fileEdit.sync_errors = data.errors;
            fileEdit.sync_warnings = data.warnings;
        },
```

I believe this is equivalent to the above pattern, and might be cleaner?

Reply:
Resolved above.